routers/user/home.go

```diff
@@ -269,6 +266,22 @@ func Issues(ctx *context.Context) {
 	}
 	opts.LabelIDs = labelIDs
 
+	repoIDStrings := ctx.QueryStrings("repos[]")
+	var repoIDs []int64
+	for _, IDString := range repoIDStrings {
+		IDint64, err := strconv.ParseInt(IDString, 10, 64)
+		if err == nil {
+			repoIDs = append(repoIDs, IDint64)
+		}
+	}
+
+	if len(repoIDs) > 0 {
+		if len(repoIDs) == 1 {
+			repoID = repoIDs[0]
```

Comment:
If I understand it correctly, `repoID` is immediatly overwritten by the `for` loop down below, so no need to set it here.

Reply:
Where do you mean? I'm not finding any place where repoID is set other than here, though maybe I'm missing something.

Reply:
The `for` loop in line 292.

Reply:
Fair enough; I've read that a local scope only for the loop, a "for repoIDs in counts, do:", rather than an explicit overwrite, but as I am uncertain I will check for necessity when I get the chance. (It's been a while since I brushed up on my Go)

Reply:
I'm not entirely convinced, I tried to recreate the situation in [The Go Playground](https://play.golang.org/p/xJuu6z9FHPF), and from what I can gather repoID is unaffected by the for-loop. I have removed the lines in question, and it doesn't seem to affect functionality so it may very well be the correct choice, but I don't want to lay the grounds for a future bug either.

Reply:
I think you're right about the loop. However it looks like removing the line in question doesn't have an effect, I'm ok with merging this.